src/shared/logs-show.c

```diff
@@ -453,6 +453,29 @@ static int output_timestamp_realtime(
         return (int) strlen(buf);
 }
 
+static void update_display_timestamps_and_bootid(sd_journal *j, dual_timestamp *ret_display_ts, sd_id128_t *ret_boot_id, char *realtime, size_t realtime_len, char *monotonic, size_t monotonic_len) {
```

Comment:
Please wrap such long line. Please see our coding style rule (https://systemd.io/CODING_STYLE/).

Please constify the inputs. `const char *realtime` and so on.